scripts/check_imports_and_dependencies.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""Check aea dependencies."""
+import copy
+import dis
+import importlib
+import re
+import sys
+from collections import defaultdict
+from functools import wraps
+from pathlib import Path
+from typing import Any, Callable, Dict, Generator, List, Set, Tuple, Union
+
+from pip._internal.commands.show import search_packages_info  # type: ignore
+
+
+AEA_ROOT_DIR = Path(__file__).parent.parent
+
+sys.path.append(str(AEA_ROOT_DIR))
+
+from aea.crypto.registries import (  # noqa # pylint: disable=wrong-import-position
+    crypto_registry,
+)
+
+
+DEP_NAME_RE = re.compile(r"(^[^=><\[]+)", re.I)  # type: ignore
+
+
+def list_decorator(fn: Callable) -> Callable:
+    """Wraps generator to return list."""
+
+    @wraps(fn)
+    def wrapper(*args, **kwargs) -> List[Any]:
+        return list(fn(*args, **kwargs))
+
+    return wrapper
+
+
+class DependenciesTool:
+    """Tool to work with setup.py dependencies."""
+
+    @staticmethod
+    def get_package_files(package_name: str) -> List[str]:
+        """Get package files list."""
+        packages_info = list(search_packages_info([package_name]))
+        if len(packages_info) == 0:
+            raise Exception(f"package {package_name} not found")
+        files = packages_info[0]["files"]
+        location = packages_info[0]["location"]
+        return [str(Path(location) / i) for i in files]
```

Comment:
So basically we solved the problem `pypi-dependency-name != package-name` through `search_packages_info`